src/main/java/ch/njol/skript/expressions/ExprSaturation.java

```diff
@@ -39,19 +36,29 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Getter;
 import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
 
 /**
  * Used to set saturation of players. Number is used in case something changes in future...
  * @author bensku
+ * 
+ * Cleaned up by LimeGlass (2.2-dev35)
  */
 @Name("Saturation")
-@Description("The saturation of a player.")
-@Examples({"saturation of player is 20 #Not hungry!"})
-@Since("2.2-Fixes-V10")
+@Description("The saturation of player.")
+@Examples("set saturation of player to 20 #Full hunger")
+@Since("2.2-Fixes-v10, 2.2-dev35 (Made 'saturation' work in player events & changers added)")
 public class ExprSaturation extends PropertyExpression<Player, Number> {
+
 	static {
-		Skript.registerExpression(ExprSaturation.class, Number.class, ExpressionType.SIMPLE, "[the] saturation [of %players%]");
+		Skript.registerExpression(ExprSaturation.class, Number.class, ExpressionType.PROPERTY, "[the] saturation [of %players%]", "%players%['s] saturation");
```

Comment:
As you have been told before, don't use `['s]` as Skript uses `[']s` normally. Whether it's proper grammar or not doesn't matter, it's more of a consistency thing right now.

If you want change this, do it in another PR.

Reply:
Correction : [Skript uses `'[s]`](https://github.com/bensku/Skript/blob/master/src/main/java/ch/njol/skript/expressions/base/PropertyExpression.java#L53)

Reply:
I hate that so much

Reply:
I don’t understand why you hate it so much, it makes it possible to write the syntax grammatically correct when you give a plural expression.

Reply:
It's not grammatically correct.

Reply:
`all players’ saturations`, go ask that English professor that you mentioned in another PR.

Reply:
That's so gross. Should be `saturation of all players`

Reply:
So you hate proper grammar :^) ?

Jokes aside, I would say `['][s]` is better for the following reasons : 
 * It doesn't break scripts that use `'[s]`
 * It still allows for proper grammar
 * It allows scripts to be a little bit less verbose (thus more concise)

Reply:
It is still correct and even if you don't like it, I'm against changing it unless it's changed for all the expressions.

Reply:
Well can't fix them all until #1132 is finished.

Reply:
Even if you do, it'd be in another PR so either do what @Syst3ms said or change it to the Skript standard.

Reply:
There's no matter of fixing, since nothing is broken. You may not like stuff like `the players'`, but it's still correct. 
Could we rather settle on `['][s]` ? That way everybody's happy.

Reply:
No. We use proper grammar whenever possible. Besides, more optional elements could slow down parsing.

If you do not this is grammatically correct, please prove it by citing a trustworthy source.

Reply:
he mean `'[s]`

Reply:
@bensku You missed a word somewhere

Reply:
That is correct grammar.

Singular: <code>player's item</code> = one item owned by one player
Plural: <code>players' item</code> = one item owned by multiple players

For reference: https://en.oxforddictionaries.com/punctuation/apostrophe#apostrophes_showing_possession

Reply:
then you can use double s